rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -446,6 +495,24 @@ private long setCorrectEndBlock(long endBlock, long fileLength, String remotePat
     return endBlock;
   }
 
+  /**
+   * Get the current size of the data cached to this system.
+   *
+   * @return The size of the cache in MB.
+   */
+  private int getCacheSizeMB()
```

Comment:
Can we use `getUsedSpaceMB` to calculate the cache space used ? This is used for eviction